apps/video/test/marionette/info_view_test.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var Video = require('./lib/video'),
+    assert = require('assert'),
+    Video_Options_View = require('./lib/video_options_view.js'),
+    Video_Info_View = require('./lib/video_info_view.js');
+
+marionette('video info view', function() {
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true
+      }
+    },
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var app, selectors, optionsView, infoView;
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+    
+    // Add file into the videos directory
+    client.fileManager.add({
+      type: 'videos',
+      filePath: 'test_media/Movies/elephants-dream.webm'
+    });
+
+    selectors = Video.Selector;
+    app = new Video(client);
+    optionsView = new Video_Options_View(client);
+    infoView = new Video_Info_View(client);
+  });
+
+  test('open video and check video info', function() {
+    // tap thumbnail to open video
+    app.launch();
+    client.helper.waitForElement(selectors.thumbnail);
+    app.waitForThumbnails(1);
+    app.playVideoFromGalleryView();
```

Comment:
nit: I think it's more intuitive to use 'app.tapThumbnail(0)' here. FWIW, I would like to rename 'playVideoFromGalleryView' to 'playVideo' or 'playFirstVideo' but that is for another bug.